ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -0,0 +1,634 @@
+
+    - **Error**  
+    Select this option to display the processes that have given an error (a process action has given an error).
+
+    - **Done**  
+    Select this option to display the processes that have been completely executed (end place has been reached).  
+
+    - **Dead**  
+    Select this option to display the processes that have automatically stopped due to a missing input. A dead process cannot be retried. It can only be aborted, that is, manually ended, and reinitiated.
+
+    - **Suspended**  
+    Select this option to display the processes that have been manually paused. The remaining process actions are temporarily stopped.
+
+    - **Aborted**  
+    Select this option to display the processes that have been manually ended. All process actions currently running are stopped and no longer executed.
+
+
+- *Workflow*  
+Click the drop-down list to display a specific workflow type. The workflows created in the *Workflows* menu entry are listed here. For detailed information about creating a workflow, see [Create a workflow](/ActindoWorkFlow/Operation/01_ManageWorkflows.md#create-a-workflow).
+
+
+
+- ![Search](/Assets/Icons/Search.png "[Search]") (Search)  
+    Click this button to display the search bar and search for a workflow.
+
+- ![Refresh](/Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
+    Click this button to update the list of workflows.
+
+- ![Columns](/Assets/Icons/Columns.png "[Columns]") Columns (x)  
+    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The x indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](/Assets/Icons/Filter.png "[Filter]") Filter (x)  
+    Click this button to display the filter bar and customize the active filters. The x indicates the number of filters that are currently active.
+
+
+The list displays all processes. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+  - [x]  
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all processes in the list are selected.
+
+  Depending of the process status, the editing toolbar may display different buttons.
+
+  - ![Abort](/Assets/Icons/Stop.png "[Abort]") (Abort)  
+    Click this button to end the process. This button is displayed when the process status is *Error* or *In Progress*. When clicking this button, the process status changes to *Aborted*.
+  - ![Suspend](/Assets/Icons/Pause.png "[Suspend]") (Suspend)  
+    Click this button to suspend the process temporarily. This button is displayed when the process status is *In Progress*.  When clicking this button, the process status changes to *Suspended*.
+  - ![Start](/Assets/Icons/Play.png "[Start]") (Start)  
+    Click this button to resume the process. This button is displayed when the process status is *Suspended*. When clicking this button, process status changes again to *In Progress*.
```

Comment:
Abschnitt in Zeile 84/85 verschieben (zwischen Filter button und "The list displays all processes...")